reference-architecture/aws-ansible/README.md

```diff
@@ -9,20 +9,33 @@ The repository contains Ansible playbooks which deploy 3 Masters in different av
 ## Prerequisites
 A registered domain must be added to Route53 as a Hosted Zone before installation.  This registered domain can be purchased through AWS.
 
-### OpenShift Playbooks
+### Deploying OpenShift Container Platform
```

Comment:
I"m from WV. I don't do English well so I will take either way with or without the "the"